transport/tchannel/inbound_test.go

```diff
@@ -94,15 +93,14 @@ func TestInboundSubServices(t *testing.T) {
 	})
 
 	require.NoError(t, i.Start())
-	require.NoError(t, itransport.Start())
-
-	chservEndpoint := itransport.ch.PeerInfo().HostPort
+	require.NoError(t, it.Start())
 
-	otransport, err := NewTransport(ServiceName("caller"))
+	ot, err := NewTransport(ServiceName("caller"))
 	require.NoError(t, err)
-	o := otransport.NewSingleOutbound(chservEndpoint)
-
+	o := ot.NewSingleOutbound(it.ListenAddr())
 	require.NoError(t, o.Start())
+	require.NoError(t, ot.Start())
```

Comment:
This fixes a tricky bug, one we might have to think about surfacing better. Calls time out if you don’t start the transport now.